packages/chevrotain/test/full_flow/error_recovery/switch_case/swithcase_recovery_spec.ts

```diff
@@ -259,38 +259,52 @@ describe("Error Recovery switch-case Example", () => {
       // case "Terry" : return 2;
       createRegularToken(CaseTok),
       createRegularToken(StringTok, "Terry"),
+      createRegularToken(ColonTok), // <-- the redundant token to be deleted
       createRegularToken(ColonTok),
       createRegularToken(ReturnTok),
       createRegularToken(IntTok, "2"),
       createRegularToken(SemiColonTok),
-      // case "Robert" : return 4;
-      createRegularToken(CaseTok),
-      createRegularToken(StringTok, "Robert"),
-      createRegularToken(ColonTok),
-      createRegularToken(ReturnTok),
-      createRegularToken(IntTok, "4"),
-      createRegularToken(SemiColonTok),
-      // case "Brandon" : return 6;
+      createRegularToken(RCurlyTok)
+    ]
+
+    const parser = new SwitchCaseRecoveryParser()
+    parser.input = input
+
+    const parseResult = parser.switchStmt()
+    expect(parser.errors.length).to.equal(1)
+    expect(parseResult).to.deep.equal({ Terry: 2 })
+  })
+
+  it("can disable single token deletion recovery", () => {
+    const input = [
+      // switch (name) {
+      createRegularToken(SwitchTok),
+      createRegularToken(LParenTok),
+      createRegularToken(IdentTok, "name"),
+      createRegularToken(RParenTok),
+      createRegularToken(LCurlyTok),
+      // case "Terry" : return 2;
       createRegularToken(CaseTok),
-      createRegularToken(StringTok, "Brandon"),
+      createRegularToken(StringTok, "Terry"),
+      createRegularToken(ColonTok), // <-- the redundant token
       createRegularToken(ColonTok),
       createRegularToken(ReturnTok),
-      createRegularToken(IntTok, "6"),
+      createRegularToken(IntTok, "2"),
       createRegularToken(SemiColonTok),
-      createRegularToken(SemiColonTok), // <-- the redundant token to be deleted
       createRegularToken(RCurlyTok)
     ]
 
     const parser = new SwitchCaseRecoveryParser()
     parser.input = input
 
+    // disable the single token deletion recovery explicitly
+    parser.singleTokenDeletionEnabled = false
+
     const parseResult = parser.switchStmt()
+    console.log(parser.errors)
```

Comment:
please remove the tracing log to avoid spam in the terminal

Reply:
Uuups.